client/rest-high-level/src/main/java/org/elasticsearch/client/ml/GetDataFrameAnalyticsStatsRequest.java

```diff
@@ -0,0 +1,76 @@
+/*
+ * Licensed to Elasticsearch under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.elasticsearch.client.ml;
+
+import org.elasticsearch.client.Validatable;
+import org.elasticsearch.client.ValidationException;
+import org.elasticsearch.client.ml.job.util.PageParams;
+import org.elasticsearch.common.Nullable;
+
+import java.util.Objects;
+import java.util.Optional;
+
+/**
+ * Request to get data frame analytics stats
+ */
+public class GetDataFrameAnalyticsStatsRequest implements Validatable {
```

Comment:
This request supports a comma delimited list of id expressions (similar to `GetDataFrameAnalyticsRequest`). Thus, it should have a similar format as that class.

Reply:
Done.

Reply:
On the second look, I cannot find the server-side logic for comma-delimited ids. Could you point me at this place? I've looked here: org.elasticsearch.xpack.core.ml.action.GetDataFrameAnalyticsStatsAction.Request.